luaui/Widgets/cmd_commandq_manager.lua

```diff
@@ -1,19 +1,19 @@
 function widget:GetInfo()
-    return {
-        name = "Command Queue Manager",
-        desc = "Skips current command or cancels the last command in command",
-        author = "[DE]LSR",
-        date = "5 Apr, 2022",
-        license = "GNU GPL, v2 or later",
-        layer = 1, --  after the normal widgets
-        enabled = true --  loaded by default?
-    }
+	return {
+		name = "Command Queue Manager",
+		desc = "Skips current command or cancels the last command in command",
+		author = "[DE]LSR",
+		date = "5 Apr, 2022",
+		license = "GNU GPL, v2 or later",
+		layer = 1, --  after the normal widgets
+		enabled = true, --  loaded by default?
+	}
 end
 
 -- Handlers
 function widget:Initialize()
-    widgetHandler:AddAction("command_skip_current", SkipCurrentCommand)
-    widgetHandler:AddAction("command_cancel_last", CancelLastCommand)
+	widgetHandler:AddAction("command_skip_current", SkipCurrentCommand, nil, "p")
```

Comment:
For some reason it was performing both 'text' and 'press' actions (2 executions per press) and still worked